build.gradle

```diff
@@ -17,7 +17,7 @@
 
 apply plugin: 'scala'
 
-version = '0.6.3-2'
+version = '0.6.2.3'
```

Comment:
@snappy-sachin As mentioned by @ashetkar the diff command above shows none of the above changes, so this PR can be closed and the jj branch deleted.